Cargo.toml

```diff
@@ -50,6 +50,7 @@ tracing = "0.1"
 tracing-subscriber = "0.3"
 url = { version = "2.4", features = ["serde"] }
 walkdir = "2"
+sha2 = "0.10.7"
```

Comment:
sha _would_ be a dev dependency btw – but #38 relies on ring so if you use ring we could keep it a direct dep